### ISSUE TITLE
Describe use case and size/performance considerations

### ISSUE THREAD

**Issue by xrotwang** [tresoldi/staticcldf#4]
Serving CLDF datasets using the [datasette plugin](https://github.com/cldf/datasette-cldf) has the nice advantage of scalability. So it would be nice to have a description of when exactly this tool is suitable (and at what size of dataset it might break down).

**Comment by tresoldi** [comment]
Yes, and also to whom/which cases it is intended. It should also be clear that automatic is deployment is possible but not necessary or mandatory.
